site/installing.md

```diff
@@ -154,81 +141,20 @@ After it’s been launched, open your browser to `http://localhost:4040`.
-
-**2.** Decide on which format version to use:
-
-**Docker Compose Format Version 1:**
-
-    probe:
-      image: weaveworks/scope:1.7.2
-      net: "host"
-      pid: "host"
-      privileged: true
-      labels:
-        - "works.weave.role=system"
-      volumes:
-        - "/var/run/docker.sock:/var/run/docker.sock:rw"
-      command:
-        - "--probe.docker=true"
-        - "--service-token=${SCOPE_SERVICE_TOKEN}"
-
-**Docker Compose Format Version 2:**
-
-    version: '2'
-    services:
-      probe:
-        image: weaveworks/scope:1.7.2
-        network_mode: "host"
-        pid: "host"
-        privileged: true
-        labels:
-          - "works.weave.role=system"
-        volumes:
-          - "/var/run/docker.sock:/var/run/docker.sock:rw"
-        command:
-          - "--probe.docker=true"
-          - "--service-token=${SCOPE_SERVICE_TOKEN}"
-
-Version 2 of this YAML file supports a network mode that may be required with any Docker plugins that you are using. See [Compose File Reference](https://docs.docker.com/compose/compose-file/) for more information.
-
-Copy the contents of one of the above fragments into a file `docker-compose.yml`.
-
-**3.** Once you've set up a compose file, launch Scope onto your local machine:
-
-    SCOPE_SERVICE_TOKEN=<token>  docker-compose up -d
-
-Where,
-
-* `SCOPE_SERVICE_TOKEN=<token>` is the token you obtained after you signed up for Weave Cloud.
-
-Note that you will need to launch Scope on every node that you want to monitor.
+## <a name="orchestrators"></a>Installing on Orchestrators
 
-**4.** Go to [https://cloud.weave.works](https://cloud.weave.works) and click 'View Instance'.
+### <a name="k8s"></a>Kubernetes
 
-## <a name="k8s"></a>Installing Scope on Kubernetes
+If your cluster is on GKE, you first need to grant the requisite
+permissions for installation:
 
-### <a name="k8s-weave-cloud"></a>With Weave Cloud (recommended)
+    kubectl create clusterrolebinding "cluster-admin-$(whoami)" --clusterrole=cluster-admin --user="$(gcloud config get-value core/account)"
 
-Weave Cloud hosts the Scope UI for you, provides secure access control for your team and saves resources such as CPU or memory usage.
-
-Sign up for a [Weave Cloud account](https://cloud.weave.works/) and obtain a token. Replace `<token>` with your token by running this on the master node or on whatever machine that has `kubectl` configured to authenticate to your Kubernetes cluster:
-
-    kubectl apply -f "https://cloud.weave.works/k8s/scope.yaml?service-token=<token>&k8s-version=$(kubectl version | base64 | tr -d '\n')"
-
-**SECURITY NOTE: This allows control of your Kubernetes cluster from Weave Cloud, which is a hosted service.**
-
-### <a name="k8s-standalone"></a>Without Weave Cloud (run Scope in standalone mode)
-
-The simplest way to get the latest release of Scope deployed onto a Kubernetes cluster is by running the following:
+To install Weave Scope on your Kubernetes cluster, run
```

Comment:
Then install Weave Scope to your cluster:

Reply:
The paragraph preceding it contains instructions that only apply to GKE. So anybody not on GKE would skip it and hence go straight to this, in which case reading "_Then_ ..." isn't right.